src/main/java/io/github/linead/nametags/domain/Attendee.java

```diff
@@ -37,11 +42,21 @@ public String getPictureUrl() {
         return pictureUrl;
     }
 
-    public void setPictureUrl(String pictureUrl) {
-        this.pictureUrl = pictureUrl;
+    public void setPictureUrl(String pictureUrl) { this.pictureUrl = pictureUrl; }
```

Comment:
I was just keeping the standard. Refer to the other beans. None the less, updated as requested.